tools/pipelines/templates/1ES/include-build-lint.yml

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+# include-build-lint template for the Build and Lint step in client build and test stability pipeline
+
+parameters:
+- name: taskBuild
```

Comment:
Feel free to file a separate task if you would prefer, but this seems like a good opportunity for us to add some documentation to our shared templates. In particular:
1. Top-level summary of what the template does, including any external prerequisites for use (particular environmental requirements, variables that are assumed to be set, etc.)
2. Add comments to each of the parameters explaining what it does